browser/src/Searchbox.tsx

```diff
@@ -76,14 +76,14 @@ export default withRouter((props: any) => {
         onChange={(e: any) => {
           setSearchDataset(e.target.value)
           if (innerSearchbox.current) {
-            ;(innerSearchbox.current as any).updateResults()
+            ; (innerSearchbox.current as any).updateResults()
           }
         }}
       >
-        <option value="gnomad_r3">gnomAD v3.1.2</option>
-        <option value="gnomad_r2_1">gnomAD v2.1.1</option>
-        <option value="gnomad_sv_r2_1">gnomAD SVs v2.1</option>
-        <option value="exac">ExAC</option>
+        <option value="gnomad_r3">gnomAD v3.1.2 (GRCh38)</option>
+        <option value="gnomad_r2_1">gnomAD v2.1.1 (GRCh37)</option>
+        <option value="gnomad_sv_r2_1">gnomAD SVs v2.1 (GRCh37)</option>
+        <option value="exac">ExAC (GRCh37)</option>
```

Comment:
I wonder if having these present in the Searchbox is a bit redundant and too much information.

I am completely on board with the genome build in the `DatasetSelector`, since that is a custom component and can be styled. I think the UI choices for that component (smaller font, grey font color) do a good job at conveying the information while keeping things legible.

Here, since it's just the HTML element, the genome build is way more prominent, and on a few pages, fairly redundant (see screenshots at the end of this comment)

I think in an ideal world, mimicking the `DatasetSelector` component to make a custom component for the `Searchbox` to both make the UI more cohesive, and allow for the genome build to show up only on dropdown, would be ideal. Though, I know that's a bigger lift than this ticket initially suggested. I'm curious as to other opinions here, as I'm a bit biased since I've had a few rounds of comments from people that have made me try to steer clear of redundancy.

---

- Variant pages have the build in parentheses in the header
  - ![Screenshot 2023-08-10 at 17 59 02](https://github.com/broadinstitute/gnomad-browser/assets/59549713/302b3f0d-f138-4094-9b6b-c6d9b4ffc1c3)


- Gene pages have the genome build present in the top left
	- ![Screenshot 2023-08-10 at 17 58 52](https://github.com/broadinstitute/gnomad-browser/assets/59549713/71e43239-7d62-4fbd-841e-0475001f0d26)